lib/perl/centreon/script/centcore.pm

```diff
@@ -827,10 +827,13 @@ sub initEngine($$$){
                 'using remote server "' . $remote_server->{name} . '" (' . $remote_server->{id} . ')'
             );
         } else {
-            $cmd = "$self->{ssh} -p $port " . $conf->{ns_ip_address} . " $self->{sudo} $self->{service} "
-                . $conf->{init_script} . " " . $options;
+            $cmd = '';
+            if ($conf->{localhost} == 0) {
```

Comment:
Maybe just add a comment on this case, the "what" is not obvious outside of this PR context